docker/incremental_load.sh.tpl

```diff
@@ -72,6 +72,17 @@ function tag_layer() {
   fi
 }
 
+function read_variables() {
+  local file=${RUNFILES}/$1
+  local new_file=$(mktemp)
+
+  sed -E "s/^([^ ]+) (.*)\$/export \\1='\\2'/g" < ${file} > ${new_file}
```

Comment:
Instead of that tricky sed, you could read the file and emit export.
```bash
while IFS='' read -r line || [[ -n "$line" ]]; do
    export "${line%% *}=${line#* }"
done < "${file}"
```

See https://stackoverflow.com/questions/10929453/read-a-file-line-by-line-assigning-the-value-to-a-variable